CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+
+# dbt_pinterest v0.8.0
+[PR #26](https://github.com/fivetran/dbt_pinterest/pull/26) introduces the following changes:
+## Pinterest API v5 Updates
+- Following Pinterest Ads deprecating the v4 API on June 30, 2023 in place of v5, the Pinterest Ads Fivetran connector now leverages the Pinterest v5 API. The following fields have been deprecated/ introduced:
+
+| **Model** | **Removed**  | **New**   |
+|---|---|---|
+|  [pinterest_ads__advertiser_report](https://fivetran.github.io/dbt_pinterest/#!/model/model.pinterest.pinterest_ads__advertiser_report) | `billing_type`, `status`  |   |
+
+- In the v5 upgrade, `advertiser_id` has been replaced by `ad_account_id`. However, to keep our Pinterest Ads package standard with our other ad packages, we have kept it as `advertiser_id`.
```

Comment:
Same comment from the source. I think this should be reworded to be more in line with the updates applied.

Reply:
updated!